pocs/camera/__init__.py

```diff
@@ -12,6 +12,7 @@
 
 from pocs.camera.camera import AbstractCamera  # pragma: no flakes
 from pocs.camera.camera import AbstractGPhotoCamera  # pragma: no flakes
+from pocs.camera.simulator import ccd as simulator_sdk
```

Comment:
My concern here is that it will be importing this for every camera that is used. Can we put this in `huntsman-pocs` or is it necessary here?

Reply:
Good point. I'd say there probably ought to some way to instantiate the SDK camera simulators from a config file in POCS itself though.

Reply:
Tricky to do this in `huntsman-pocs`. The problem is that camera creation in both `POCS` _and_ `huntsman-pocs` assumes **all** `Camera` classes are `pocs.camera.<model name>.Camera`.

To work around this in `huntsman-pocs` you'd need to do something like 
```
import pocs.camera.simulator.ccd as pocs.camera.simulator_sdk
```
but that's illegal syntax. Would have to start either manually fiddling with the `pocs.camera` namespace, or special casing the SDK camera simulator so it doesn't have to be in `pocs.camera`.  Neither option seems very appealing.

Reply:
What I could do is basically do what the DSLR simulator is currently doing. Move `ccd.py` out of `pocs/camera/simulator` into it's own directory and import the `Camera` class in the `__init__.py`. Means we get two simulated camera directories.